src/map/conquest_system.cpp

```diff
@@ -416,13 +418,17 @@ namespace conquest
         zoneutils::ForEachZone([ranking, isConquestAlliance](CZone* PZone)
         {
             // only find chars for zones that have had conquest updated
-            if (PZone->GetRegionID() <= REGION_TYPE::DYNAMIS)
+            REGION_TYPE regionId = PZone->GetRegionID();
+            if (regionId <= REGION_TYPE::TAVNAZIA)
             {
                 uint8 influence = conquest::GetInfluenceGraphics(PZone->GetRegionID());
                 uint8 owner     = conquest::GetRegionOwner(PZone->GetRegionID());
-
                 luautils::OnConquestUpdate(PZone, Conquest_Tally_Start, influence, owner, ranking, isConquestAlliance);
             }
+            else if (regionId > REGION_TYPE::TAVNAZIA && regionId < REGION_TYPE::DYNAMIS)
+            {
+                luautils::OnCityConquestUpdate(PZone, Conquest_Tally_Start, ranking, isConquestAlliance);
```

Comment:
We want to call certain city logic for conquest updates, but cities don't have owner / influence data as they are not conquest zones per se. I think this is the cleanest way to handle this. Even though its more boilerplate, it does no longer call the same method for zones that have different assumptions of owner / influence values, which could potentially cause issues.